tests/components/wolflink/test_config_flow.py

```diff
@@ -0,0 +1,139 @@
+"""Test the Wolf SmartSet Service config flow."""
+from httpcore import ConnectError
+from wolf_smartset.models import Device
+from wolf_smartset.token_auth import InvalidAuth
+
+from homeassistant import config_entries, data_entry_flow, setup
+from homeassistant.components.wolflink.const import (
+    DEVICE_GATEWAY,
+    DEVICE_ID,
+    DEVICE_NAME,
+    DOMAIN,
+)
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+
+CONFIG = {
+    DEVICE_NAME: "test-device",
+    DEVICE_ID: 1234,
+    DEVICE_GATEWAY: 5678,
+    CONF_USERNAME: "test-username",
+    CONF_PASSWORD: "test-password",
+}
+
+INPUT_CONFIG = {
+    CONF_USERNAME: CONFIG[CONF_USERNAME],
+    CONF_PASSWORD: CONFIG[CONF_PASSWORD],
+}
+
+DEVICE = Device(CONFIG[DEVICE_ID], CONFIG[DEVICE_GATEWAY], CONFIG[DEVICE_NAME])
+
+
+async def test_show_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+    assert result["step_id"] == "user"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5872960)